src/constants/translations/en/index.js

```diff
@@ -23,7 +23,7 @@ import completeProfile from './complete-profile.json'
 import tutorProfilePage from './tutor-profile-page.json'
 import adminTable from './admin-table.json'
 import constant from './constant.json'
-import findOffers from './find-offer-page.json'
+import findOffer from './find-offer-page.json'
 import filters from './filters.json'
 
 const en = {
```

Comment:
Are you sure that you fixed an error in the correct place? 
I think it is wrong because you fixed only your problem but broke down in another place.
Check it, please)
Try will fix here your problem and don't change the name of this import "findOffers"
![image](https://user-images.githubusercontent.com/83120263/228455219-022ef83e-4a8d-4f71-9891-d17b6963cfaf.png)

Reply:
fixed